app/main.py

```diff
@@ -1,3 +1,48 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+
+from app.customer import Customer
+from app.car import Car
+from app.shop import Shop
+
+
+def shop_trip() -> None:
+    with open("app/config.json", "r") as file_open:
+        data = json.load(file_open)
+
+    customers = [Customer(customer) for customer in data["customers"]]
+    shops = [Shop(shop) for shop in data["shops"]]
+
+    for i in range(len(customers)):
+        customer = customers[i]
```

Comment:
is it the same as `for customer in customers`?

Reply:
Yes:) I have changed it) thank you)